test/cri-containerd/runpodsandbox_test.go

```diff
@@ -273,6 +292,25 @@ func Test_RunPodSandbox_PhysicalMemory_LCOW(t *testing.T) {
 	runPodSandboxTest(t, request)
 }
 
+func Test_RunPodSandbox_FullyPhysicallyBacked_LCOW(t *testing.T) {
```

Comment:
Maybe a test that execs and sees that we are actually physically backed and have zero vpmem devices. The first part would be handled by using the test functions in here https://github.com/microsoft/hcsshim/pull/809 but the zero vpmem I think we could just 'fdisk -l' and string search for pmem. WCOW would only need the first half

Reply:
That PR doesn't explicitly check that the pod is physically backed, it just checks that the working set is as expected for a physically backed pod. I don't think that would be an appropriate way to actually check that the pod is backed correctly here.

Reply:
What other method would we have? If it's PA backed and we get anything other than the amount allocated to the UVM is this not a good indicator that we're not PA backed? That seems to be the most straightforward.

Reply:
We don't have to add this test here it was more of a 'might be a nice addition'. Could make this down the road doesn't have to be in this PR

Reply:
I don't think that's a good way of testing this because the test you linked tests a specific scenario: PA backed uvm working set stats information. If that's ever broken somehow, that would cause a theoretical test here relying on it to break even though nothing is actually wrong with backing a UVM with physical memory.

Reply:
That's fair. Don't know how else to test this off the top of my head

Reply:
I'm not sure these tests here are really meant to test the annotation's feature as opposed to that the pod runs anyways.

Reply:
I guess this scenario will automatically be tested when we have actual tests that add devices to the pod that is PA backed so there is no need to verify if this pod is actually PA backed in these tests, right?